src/app/global_styles.tsx

```diff
@@ -74,8 +74,18 @@ const materialTheme = (chosenTheme: DefaultTheme): Theme =>
                     root: {
                         transition: "none",
                         textTransform: "none",
+                        "&:hover": {
+                            opacity: 0.7,
+                        },
+                        "&:focus": {
+                            opacity: 0.7,
+                            boxShadow: `0px 0px 0px 2px ${chosenTheme.primary.foreground[0]}`,
```

Comment:
`boxShadow` feels a bit strange here, is there a reason for using it over `border`?